tony-history-server/test/utils/TestHdfsUtils.java

```diff
@@ -27,6 +28,71 @@ public static void setup() {
     }
   }
 
+  @Test
+  public void testDeleteMultiDir() throws IOException {
+    Path deleteDirs = new Path("./test/resources/deleteDirs");
+    for (int i = 0; i < 5; i++) {
+      fs.mkdirs(new Path(deleteDirs, "dir" + i));
+    }
+    FileStatus[] statuses = fs.listStatus(deleteDirs);
+
+    HdfsUtils.deleteMultiDir(fs, statuses);
+    assertEquals(0, fs.listStatus(deleteDirs).length);
```

Comment:
Before the `deleteMultiDir` line, I would suggest adding an `assertEquals(5, fs.listStatus(deleteDirs).length);` check just as a sanity that 5 directories were created successfully before we deleted them.

Reply:
Fixed.